### ISSUE TITLE
Question: custom loss with additional data

### ISSUE THREAD

**Issue by fabz1** [manujosephv/pytorch_tabular#159]
Hello, I would like to use additional data (e.g. a feature) as part of a custom loss function. Is there an easy way to achieve something like the following or do I need to adjust the data loader?:

def custom_loss(y_pred, y_true, data_series):
    # Loss calculation

Thanks!

**Comment by manujosephv** [comment]
Interesting... right now the custom loss just supports `y_pred` and `y_true` because that is what is used internally to call the loss function..  If the `data_series` is static, you can still hack it by using a `partial` function.. If it is dynamic, then it becomes more complicated. Maybe some complicated setup with callbacks or something might work, but I haven't explored that myself...
